### ISSUE TITLE
Error 500 when "from" > "to"

### ISSUE THREAD

**Issue by kalkov** [santiment/sanbase2#970]
While playing with the GraphQL API I have accidentally set datetime `from` param to be after the `to` datetime param : 

```graphql
{
      historyPrice(slug: "bitcoin", 
                   from: "2019-02-01T00:00:00Z",
                   to: "2019-01-29T00:00:00Z", 
                   interval: "1d") {
        datetime
        priceUsd
        volume      
  }
}
```

The result was an error with code 500. Would be nice to return a human readable error message and more suitable error code.

**Comment by IvanIvanoff** [comment]
@tspenov @kalkov @valo @Partysun @DmitriVanGuard This PR actually changes the behavior quite a lot.
Previously when the `from` and `to` were both outside the allowed interval it was just rewriting the params to some wrong params - `to` was before `from` most probably.

Now it will stop early all such cases and return an error right away. We need to decide if restriction to the API is wanted and won't introduce errors.

**Comment by IvanIvanoff** [comment]
Closed by #1138
